src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -166,15 +167,16 @@ public static void clearCaches() {
 
     @Deprecated
     public BitbucketCloudApiClient(boolean enableCache, int teamCacheDuration, int repositoriesCacheDuration,
-                                   String owner, String repositoryName, StandardUsernamePasswordCredentials credentials) {
-        this(enableCache, teamCacheDuration, repositoriesCacheDuration, owner, repositoryName,
+                                   String owner, String projectKey, String repositoryName, StandardUsernamePasswordCredentials credentials) {
```

Comment:
In this case will be original function no more reachable. I think, you shall keep the function as well and just use some default value in the call in line 171.

Reply:
Kept original deprecated constructor as suggested.